tests/backup/backup_basic_test.go

```diff
@@ -146,10 +148,89 @@ var _ = BeforeSuite(func() {
 })
 
 var _ = AfterSuite(func() {
-	StartTorpedoTest("CleanupBuckets", "Removing buckets", nil, 0)
+	StartTorpedoTest("Environment cleanup", "Removing Px-Backup entities created during the test execution", nil, 0)
 	defer dash.TestSetEnd()
 	defer EndTorpedoTest()
-	// Cleanup all bucket after suite
+
+	// Cleanup all non admin users
+	ctx, err := backup.GetAdminCtxFromSecret()
+	log.FailOnError(err, "Fetching px-central-admin ctx")
+	allUsers, err := backup.GetAllUsers()
+	dash.VerifySafely(err, nil, "Verifying cleaning up of all users from keycloak")
```

Comment:
Can you check here if these are logging to the dashboard. 
As there no testcase created here, these dash logs wont be posted to dashboard

Reply:
Yes. It is being pushed to [dashboard](http://aetos.pwx.purestorage.com/resultSet/testSetID/88569/testCaseID/348628)

Since it part of `Environment cleanup` test which is part of AfterSuite. Let me know if this is what you are looking for.